build.gradle.kts

```
@@ -400,3 +400,10 @@ tasks.register<Zip>("allIncubationReportsZip") {
 }
 
 fun Project.collectAllIncubationReports() = subprojects.flatMap { it.tasks.withType(IncubatingApiReportTask::class) }
+
+allprojects {
+    tasks.withType<AbstractArchiveTask>() {
```

Doing this instead will prevent the tasks from being realized when they aren't needed.

```suggestion
    tasks.withType<AbstractArchiveTask>().configureEach {
```